packages/api/public/spec/schemas/parameters.yml

```diff
@@ -30,3 +30,11 @@ userId:
   schema:
     type: string
   example: '123e4567-e89b-12d3-a456-426614174000'
+lemmaId:
+  name: lemmaId
+  in: path
+  description: TODO - add description here.
```

Comment:
I need a good description here. Any ideas?

Reply:
I'd just say something like `The id of the lemma as a strongs number`